libpermian/plugins/github/test.py

```diff
@@ -58,7 +58,7 @@ def setUp(self):
         self.settings = Settings(cmdline_overrides={'github': {'pull-request': '42',
                                                                'repository': 'user/test',
                                                                'token': '1234'},
-                                                    'reportSender-github-pr': {'throttleInterval': '0'}},
+                                                    'github-pr': {'throttleInterval': '0'}},
```

Comment:
`github-pr` should now be `github`. Also it seems that this test doesn't check that throttling is disabled, because this test is now using default value 60 and not failing. But this is not blocking merge.

Reply:
It probably should but such a change should imho not be in this pull request but rather in separate one focusing on renaming the reporting. This pull request is about the generic mechanism, hence removing the "reportSender-' prefix.